source/install/desktop-app-install.rst

```diff
@@ -18,159 +18,190 @@ Desktop application install guides
 
-Unofficial, community-driven ``.rpm`` packages are available.
+    3. Drag the Mattermost application to the **Applications** folder.
 
-1. Download the latest version of the Mattermost Desktop App:
+    .. tip:: 
+      You can review the current version of your Desktop App by selecting **Mattermost > About Mattermost** from the macOS menu bar. 
 
-  - 32-bit systems `mattermost-desktop-5.0.4-linux-i686.rpm <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-i686.rpm>`__
-  - 64-bit systems `mattermost-desktop-5.0.4-linux-x86_64.rpm <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-linux-x86_64.rpm>`__
+  .. tab:: Windows
 
-2. At the command line, execute one of the following commands depending on the package that you downloaded:
+    Windows 8.1+ is required. Automatic app updates are supported and enabled. When a new version of the Desktop App is released, your app updates automatically.
 
-  - 32-bit systems ``sudo rpm -i mattermost-desktop-5.0.4-linux-i686.rpm``
-  - 64-bit systems ``sudo rpm -i mattermost-desktop-5.0.4-linux-x86_64.rpm``
+    **Install the Mattermost Desktop App**
 
-3. Run Mattermost as a desktop app.
+    1. Download the latest version of the Mattermost desktop app: `32/64-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-setup-5.0.4-win.exe>`__
+    2. From the **\Downloads** folder, right-click on the file ``mattermost-desktop-setup-5.0.4-win.exe``, then select **Open** to start an installer for the app. Once finished, the Mattermost desktop app opens automatically.
 
-Arch Linux-based systems
-~~~~~~~~~~~~~~~~~~~~~~~~
+    **MSI Installer and group policies (beta)**
 
-To install the Desktop client on Arch Linux, see the `Mattermost page <https://wiki.archlinux.org/index.php/Mattermost>`__ on the Arch Linux wiki.
+    You can download the latest version of the Mattermost Desktop App MSI installer (Beta):
 
-Snapcraft package
-~~~~~~~~~~~~~~~~~
+    - MSI for `64-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-x64.msi>`__
+    - MSI for `32-bit version of Windows <https://releases.mattermost.com/desktop/5.0.4/mattermost-desktop-5.0.4-x86.msi>`__
 
-A snap is available for systems that have Snapcraft installed. Snapcraft is installed by default on Ubuntu 16.04 and later, but for most other Linux distributions you can install it manually. To install Snapcraft, see `Install snapd <https://snapcraft.io/docs/core/install>`__ on the Snapcraft website.
+    The following group policies are available:
 
-1. At the command line, execute the following command:
+    +--------------------------+------------------------------------------------------------+----------------------+------------------------+------------------+
+    | Group policy             | Description                                                | Required Mattermost  | Setting                | State options    |
```

Comment:
I would change "Required Mattermost version" to "Mattermost Release". I say this as the version is in the table.